src/Strategy/Dpkg/Dpkg.hs

```diff
@@ -0,0 +1,56 @@
+module Strategy.Dpkg.Dpkg (analyze) where
+
+import Container.OsRelease (OsInfo (OsInfo))
+import Control.Effect.Diagnostics (
+  Diagnostics,
+  context,
+ )
+import Data.String.Conversion (toText)
+import DepTypes (
+  DepType (LinuxDEB),
+  Dependency (Dependency),
+  VerConstraint (CEq),
+ )
+import Effect.ReadFS (
+  Has,
+  ReadFS,
+  readContentsParser,
+ )
+import Graphing (
+  Graphing,
+  directs,
+ )
+import Path (Abs, Dir, File, Path)
+import Strategy.Dpkg.Parser (
+  DpkgEntry (..),
+  dpkgEntriesParser,
+ )
+import Types (GraphBreadth (..))
+
+buildGraph :: OsInfo -> [DpkgEntry] -> Graphing Dependency
+buildGraph (OsInfo os osVersion) pkgs = directs (map toDependency pkgs)
```

Comment:
[nit] I think this can eta reduce:
`buildGraph (OsInfo os osVersion) = directs . map toDependency`